lib/evm_database_ops.rb

```diff
@@ -112,17 +114,18 @@ def self.restore(db_opts, connect_opts = {})
       else
         connect_opts[:remote_file_name] ||= File.basename(backup_file_name(action))
         backup_folder = action == :dump ? "db_dump" : "db_backup"
-        uri = File.join(connect_opts[:uri], backup_folder, connect_opts[:remote_file_name])
+        connect_opts[:uri] = uri = File.join(connect_opts[:uri], backup_folder, connect_opts[:remote_file_name])
```

Comment:
Will this matter for the other mount types as we can't mount a file path? We were not previously changing the value of `connection_opts[:uri]` in the caller here.

Can you ensure that you re-test this with an NFS backup to be sure we don't regress?